Bottom: b3d68627727f8c5cf7cc1b033b9fa5c00d9354a4
Top:    eb6940eb952f9cdcfb768a51e3ff38ce5f14f2a5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 02:16:54 +0100

Refresh of crosshair-c-nasty-thing-fixme

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 2d4e26d..60318a5 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -693,8 +693,10 @@ CrosshairOn (bool BlockToo)
   if (!Crosshair.On)
     {
       Crosshair.On = true;
+#if 0
       DrawAttached (BlockToo);
       DrawMark (true);
+#endif
     }
 }
 
@@ -707,8 +709,10 @@ CrosshairOff (bool BlockToo)
   if (Crosshair.On)
     {
       Crosshair.On = false;
+#if 0
       DrawAttached (BlockToo);
       DrawMark (true);
+#endif
     }
 }
